CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 Enhancements:
 - Added XML documentation to BeginScope and RequireScope lifetime extensions (@jonorossi)
 - Upgraded build to use NUnit Adapters (@fir3pho3nixx, #243)
+- Remove BOM from all source code files (@davidshen84, #405)
```

Comment:
Maybe we should say:

`Fixed encoding problems prevented build under GB2312 character encoding system`